Gemfile

```diff
@@ -92,6 +92,8 @@ gem "thin",                           "~>1.7.0",       :require => false
 gem "uglifier",                       "~>3.0.0"
 gem "websocket-driver",               "~>0.6.3"
 
+gem "elasticsearch", "~>5.0.0"
```

Comment:
This is temporary to make the test green, we should find a place for it either in the pending gems or in its own provider if we choose to split this.